demo/index.html

```diff
@@ -15,6 +15,8 @@
 	<body>
 		<d2l-demo-snippet>
 			<d2l-labs-media-player src="http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4"></d2l-labs-media-player>
+			<br>
+			<d2l-labs-media-player src="https://www.soundhelix.com/examples/mp3/SoundHelix-Song-1.mp3"></d2l-labs-media-player>
 		</d2l-demo-snippet>
```

Comment:
Typically we'd break this into two separate `<d2l-demo-snippet>`s, with a heading for each:

```suggestion
		<h2>Video Player</h2>
		<d2l-demo-snippet>
			<d2l-labs-media-player src="http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4"></d2l-labs-media-player>
		</d2l-demo-snippet>

		<h2>Audio Player</h2>
		<d2l-demo-snippet>
			<d2l-labs-media-player src="https://www.soundhelix.com/examples/mp3/SoundHelix-Song-1.mp3"></d2l-labs-media-player>
		</d2l-demo-snippet>
```